solidity/core/contracts/Inbox.sol

```diff
@@ -37,8 +37,8 @@ contract Inbox is IInbox, Version0, Common {
 
     // Domain of outbox chain
     uint32 public override remoteDomain;
-    // re-entrancy guard
-    uint8 private entered;
+    // Re-entrancy guard. Use a full slot to avoid an SLOAD before each SSTORE.
+    uint256 private entered;
```

Comment:
the [openzeppelin mixin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.6.0/contracts/security/ReentrancyGuard.sol) does this
any reason not to use that here?